docs/source/installation/micromamba-installation.rst

```diff
@@ -50,7 +50,7 @@ On Windows Powershell, use
 
 .. code :: powershell
 
-   Invoke-Expression ((Invoke-WebRequest -Uri https://micro.mamba.pm/install.ps1).Content)
+   Invoke-Expression ((Invoke-WebRequest -Uri https://micro.mamba.pm/install.ps1 -UseBasicParsing).Content)
```

Comment:
I'm not a specialist at all so I just discovered that, but `-UseBasicParsing` seems to be deprecated for Powershell >= 6
If we need to support v5 then I guess it's ok, also it's supposed to be silent with higher versions.
See: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7.4#-usebasicparsing